src/main/java/com/wepay/model/data/PhoneData.java

```diff
@@ -0,0 +1,18 @@
+package com.wepay.model.data;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class PhoneData {
```

Comment:
Looks like duplicated as `InternationalPhoneNumberData.java`
please replace it.